prusti-tests/tests/cargotest.rs

```diff
@@ -161,10 +161,11 @@ fn test_local_project<T: Into<PathBuf>>(project_name: T) {
 
     // Set the expected exit status, stdout and stderr
     let mut test_builder = project.process(cargo_prusti_path());
+    test_builder.arg("--quiet");
```

Comment:
```suggestion
    let mut test_builder = project.process(cargo_prusti_path()).arg("--quiet");
```
(If it compiles)

Reply:
This was the first thing I tried, but unfortunately that didn't compile :) The issue is that `.arg()` returns a `&mut Execs`, which is a borrow of temporary variable `project.process(cargo_prusti_path())`, which then doesn't live "long enough".